standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -9479,43 +9479,38 @@ void updateMetrics() throws MetaException {
 
   @Override
   public PrimaryKeysResponse get_primary_keys(PrimaryKeysRequest request) throws TException {
-    String catName = request.isSetCatName() ? request.getCatName() : getDefaultCatalog(conf);
-    String db_name = request.getDb_name();
-    String tbl_name = request.getTbl_name();
-    startTableFunction("get_primary_keys", catName, db_name, tbl_name);
+    if (!request.isSetCatName())
```

Comment:
Please use flower brackets, even for single statement if blocks.

Reply:
Sure. For now I have replaced it with conditional operator.